src/config/bluetooth.md

```diff
@@ -21,9 +21,9 @@ Then, add your user to the `bluetooth` group and restart the `dbus` service, or
 simply reboot the system. Note that restarting the `dbus` service may kill
 processes making use of it.
 
-> Note: To use an audio device such as a wireless speaker or headset, ALSA users
-> need to install the `bluez-alsa` package, while
-> [PulseAudio](./media/pulseaudio.md) users do not need any additional software.
+Note: To use an audio device such as a wireless speaker or headset, ALSA users
+need to install the `bluez-alsa` package, while
+[PulseAudio](./media/pulseaudio.md) users do not need any additional software.
```

Comment:
This note is unclear and probably important important enough to not be a note. "device such as" should clearly specify Bluetooth audio devices. (assuming that this applies to all bluetooth audio)

Reply:
i'm not sure what to do with this one. @void-linux/pkg-committers, any thoughts?

Reply:
yeah, probably shouldn't be a note.  I've seen a handful of questions about this in IRC over the years.

Reply:
(i'm removing the word note here)